google/api_core/exceptions.py

```diff
@@ -116,15 +119,16 @@ class GoogleAPICallError(GoogleAPIError, metaclass=_GoogleAPICallErrorMeta):
     This may be ``None`` if the exception does not match up to a gRPC error.
     """
 
-    def __init__(self, message, errors=(), response=None):
+    def __init__(self, message, errors=(), details=(), response=None):
```

Comment:
We need a corresponding update to the class docstring above.

Reply:
Done